pages/documentation/grid/model/index.md

```diff
@@ -10,150 +10,122 @@ latex: true
 
-**Bus**
-[![Javadoc](https://img.shields.io/badge/-javadoc-blue.svg)](https://javadoc.io/doc/com.powsybl/powsybl-core/latest/com/powsybl/iidm/network/Bus.html)<br>
-A bus is a set of equipments connected at the same voltage.
-When an equipment is connected to a bus, in the IIDM descrition it corresponds to a `Terminal` object.
-In IIDM there is thus one `Terminal` per connected equipment.
+The following diagram represent an example voltage level with two busbars separated by a circuit breaker, a transformer connected to one of them and three generators that can connect to any of the two busbars. The three topology levels are shown.
 
-<span class="heading" id="bb-switch"></span>**Switch**
-[![Javadoc](https://img.shields.io/badge/-javadoc-blue.svg)](https://javadoc.io/doc/com.powsybl/powsybl-core/latest/com/powsybl/iidm/network/Switch.html)<br>
-<span style="color:red"> TODO: explain the difference with node/breaker switches</span>
+![VoltageLevel](img/index/voltage-level.svg){: width="100%" .center-image}
 
-### Injections
+When defining the model, the user has to specify how the different equipment connect to the network. The connectivity level can be different in each voltage level of the model. If the voltage level is built at node/breaker level, the user has to specify a `Node` when adding an equipment to the model. If the user is building using bus/breaker level, the `ConnectableBus` of the equipment must be specified. If the equipment connection to the network is closed, then the attribute `Bus` must be set and must be equal to `ConnectableBus`. Using this information, the model creates a `Terminal` that will be used to manage the point of connection of the equipment to the network.
 
-An injection in IIDM is any AC equipment with a single connection point to a voltage level.
-Below are the different types of injections supported by PowSyBl.
 
-#### Generator 
+### Generator
 [![Javadoc](https://img.shields.io/badge/-javadoc-blue.svg)](https://javadoc.io/doc/com.powsybl/powsybl-core/latest/com/powsybl/iidm/network/Generator.html)
 
-A generator is an active equipment that injects active power, and injects or consumes reactive power. 
+A generator is an active equipment that injects active power, and injects or consumes reactive power.
 It may be controlled to hold a voltage or reactive setpoint somewhere in the network (not necessarily directly where it is connected).
 
-<span style="color:red"> TODO: add a sketch where the sign convention is indicated.</span>
-
-**Characteristics**
-
 | Attribute | Unit | Description |
 | --------- | ---- | ----------- |
-| $$MinP$$ | MW | The minimal active power |
-| $$MaxP$$ | MW | The maximum active power |
+| $$MinP$$ | MW | Minimum generator active power output |
+| $$MaxP$$ | MW | Maximum generator active power output |
+| $$ReactiveLimits$$ | Mvar | Operational limits of the generator (P/Q/U diagram) |
```

Comment:
I wonder about the use of U for the voltage as we prefer V in Powsybl.

Reply:
Done.